credential-controller.js

```diff
@@ -159,6 +159,9 @@ function factory(
         if(!('identity' in session) && self.credential.sysIsPublic) {
           // display a public credential to a non-authenticated user
           _display('credentialInfo');
+        } else if('CREDENTIAL_ADMIN' in session.identity.sysPermissionTable) {
```

Comment:
@dlongley I actually tested this where sysPermission was undefined and it behaves properly.  I also created this test: https://repl.it/CIO5

Does this address your concern?